var/spack/repos/builtin/packages/py-kosh/package.py

```diff
@@ -20,6 +20,7 @@ class PyKosh(PythonPackage):
     # notify when the package is updated.
     maintainers = ['doutriaux1']
 
+    version('2.1', sha256='597ed5beb4c3c3675b4af15ee7bfb60a463d5bda2222cd927061737ed073d562')
     version('2.0', sha256='059e431e3d3219b53956cb464d9e10933ca141dc89662f55d9c633e35c8b3a1e')
 
     depends_on('py-setuptools', type='build')
```

Comment:
Can't remember if we discussed this last time, but shouldn't it be `py-llnl-sina@1.11:` with a colon?

Reply:
@adamjstewart what would the colon do? I pushed the change anyway. Thx.